depends/patches/qt/fix_darwin_archs.patch

```diff
@@ -0,0 +1,16 @@
+To cross compile for the x86_64-darwin host on aarch64-darwin,
+passing QMAKE_APPLE_DEVICE_ARCHS=arm64 to the configure script
+does not work for some reasons.
```

Comment:
I would prefer if we could document the reason it doesn't work. And any possible information on how/when/what circumstances this patch could be dropped

Reply:
> I would prefer if we could document the reason it doesn't work.

I failed to track this bug down, unfortunately.

> And any possible information on how/when/what circumstances this patch could be dropped

As this is an ad-hoc hack, we should try to drop it while bumping Qt version in the future.

Reply:
I think the #23583 is the correct solution for this problem.